airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1003,3 +1003,73 @@ def parse_response(self, response: requests.Response, stream_slice: Mapping[str,
         response = response.json().get("workflow_runs")
         for record in response:
             yield record
+
+
+class TeamMembers(GithubStream):
+    """
+    API docs: https://docs.github.com/en/rest/reference/teams#list-team-members
+    """
+
+    primary_key = ["id", "team_slug"]
+
+    def __init__(self, parent: HttpStream, **kwargs):
+        super().__init__(**kwargs)
+        self.parent = parent
```

Comment:
as for me, I would better declare parent class directly in this class. like it is done in another streams  with 'parent_entity'

Passing 'parent' class as  an argument assumes that parent can be different.  But here we should preciscly define that we need only  Teams stream.

Reply:
yes I thought about it
but because there are more then one level of nesting of classes:

1. TeamMembers -> Teams
2. TeamMemberships -> TeamMembers -> Teams

I decided just to keep this approach
some old classed also use this approach

I changed type annotation to be more specific 
https://github.com/airbytehq/airbyte/pull/11893/commits/8411b31e0a2ee83fac1b157d1d239ba053222032